package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "mdlinks",
+  "version": "1.0.0",
+  "description": "## Preámbulo",
+  "main": "app.js",
+  "bin": {
+    "mdlinks": "cli.js"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

La sección de scripts debe contener las herramientas como Jest
```suggestion
    "test": "jest"
```